instrumentation/opentelemetry-instrumentation-elasticsearch/tests/test_elasticsearch.py

```diff
@@ -413,3 +414,18 @@ def response_hook(span, response):
             json.dumps(response_payload),
             spans[0].attributes[response_attribute_name],
         )
+
+    def test_no_op_tracer_provider(self, request_mock):
+        ElasticsearchInstrumentor().uninstrument()
+        ElasticsearchInstrumentor().instrument(tracer_provider=trace_api.NoOpTracerProvider())
+
+        request_mock.return_value = (
+            1,
+            {},
+            '{"found": false, "timed_out": true, "took": 7}',
+        )
+        es = Elasticsearch()
+        es.get(index="test-index", doc_type="_doc", id=1)
```

Comment:
I suggest validating es.get response value -> so we'll know for sure that no spans were created due to NoOpTracerProvider